tests/antithesis/Makefile

```diff
@@ -28,9 +28,17 @@ antithesis-build-etcd-image-release-3.6: antithesis-build-etcd-image
 antithesis-build-etcd-image-main: REF=main
 antithesis-build-etcd-image-main: antithesis-build-etcd-image
 
+.PHONY: antithesis-templatize-docker-compose
+antithesis-templatize-docker-compose:
+	export REF=$(REF) && cat ./docker-compose.yml | envsubst > docker-compose-config.yml
```

Comment:
Maybe we could render the docker-file when building docker image?

Reply:
Like 
```
FROM docker.io/library/ubuntu:latest as build
RUN \
  apt-get update \
  && apt-get -y install gettext-base \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
ARG REF
COPY docker-compose.yml /docker-compose.yml.template
RUN cat /docker-compose.yml.template | envsubst > /docker-compose.yml
FROM scratch
COPY --from=build /docker-compose.yml /docker-compose.yml
```

Reply:
Yes, this is a better way.  Thanks for the feedback Marek.